### ISSUE TITLE
spopt sub-sub modules not included on PyPI?

### ISSUE THREAD

**Issue by jGaboardi** [pysal/spopt#128]
Currently the PyPI published [`v0.0.3`](https://pypi.org/project/spopt/0.0.3/#files) and [`v0.0.2`](https://pypi.org/project/spopt/0.0.2/#files) do not include `region`, `locate`, or `transport`. However, the PyPI published[`v0.0.1`](https://pypi.org/project/spopt/0.0.1/#files) and the [GitHub releases](https://github.com/pysal/spopt/releases) do include them. This problem is causing the failure in conda-forge/staged-recipes#13708.

My first thought is that there is an issue with our [`release_and_publish.yml`](https://github.com/pysal/spopt/blob/main/.github/workflows/release_and_publish.yml).

See also #118

**Comment by sjsrey** [comment]
Why do we have `locate` and `transport` if those do not yet have any functionality?

**Comment by jGaboardi** [comment]
They were part of the directory structure, albeit simply place holders with `__init__.py` files for the time being. Should we remove for the time being?

**Comment by sjsrey** [comment]
I think it might confuse users and/or lead to installation mysteries of the kind we are currently trying to debug :->

**Comment by knaaptime** [comment]
Huh. I tried that same check yesterday and it worked, though I didn’t look to see which version. My best guess would be something related to manifest. Will keep digging today

**Comment by sjsrey** [comment]
Aspirational goals/plans should get moved into a roadmap document on the site

**Comment by jGaboardi** [comment]
> I think it might confuse users and/or lead to installation mysteries of the kind we are currently trying to debug :->

OK. I will remove them from the directory and create an issue to add them back later.

**Comment by sjsrey** [comment]
Testing a local build, the tarball gets:
```
 ❯ tar xzvf spopt-0.0.4.tar.gz                                           [09:27:22]
spopt-0.0.4/
spopt-0.0.4/LICENSE.txt
spopt-0.0.4/MANIFEST.in
spopt-0.0.4/PKG-INFO
spopt-0.0.4/README.md
spopt-0.0.4/requirements.txt
spopt-0.0.4/requirements_dev.txt
spopt-0.0.4/requirements_docs.txt
spopt-0.0.4/requirements_tests.txt
spopt-0.0.4/setup.cfg
spopt-0.0.4/setup.py
spopt-0.0.4/spopt/
spopt-0.0.4/spopt/BaseClass.py
spopt-0.0.4/spopt/__init__.py
spopt-0.0.4/spopt.egg-info/
spopt-0.0.4/spopt.egg-info/PKG-INFO
spopt-0.0.4/spopt.egg-info/SOURCES.txt
spopt-0.0.4/spopt.egg-info/dependency_links.txt
spopt-0.0.4/spopt.egg-info/not-zip-safe
spopt-0.0.4/spopt.egg-info/requires.txt
spopt-0.0.4/spopt.egg-info/top_level.txt
```
so something is not correct in the setup.py?

**Comment by jGaboardi** [comment]
there was no problem when I did the first manual release on PyPI or with the tagged releases here on GH. the problem was only introduced when I started using the release action and #129 may solve the problem.

**Comment by sjsrey** [comment]
Just to be clear, a `python setup.py sdist` is not grabbing any of the submodules, so I think that is part of the problem (orthogonal to the release actions)

**Comment by jGaboardi** [comment]
solved in #131.
